content/S2016/S16-event-boardgames.md

```diff
@@ -0,0 +1,22 @@
+title: Board Games Night with Grads
+Date: 2016-5-12
+Category: Events
+Tags: social, academic
+Slug: board-games-s16
+Author: Clare Ng
+Summary: Come out for a night of board games with grad students!
+
+Join us for a night of board games with some female computer science 
```

Comment:
This looks good to me on the Heroku deploy, I just had a question about this line. Did the WiCS grad committee specifically give us this description or are we making an assumption about the gender identities of the grad students that will be present? If there are e.g. non-binary grad students attending I think this description would make them uncomfortable.